models/settings_helpers.go

```diff
@@ -257,6 +264,14 @@ func UpdateSettings(q reform.DBTX, params *ChangeSettingsParams) (*Settings, err
 		settings.IntegratedAlerting.SlackAlertingSettings = params.SlackAlertingSettings
 	}
 
+	if params.DisableBackupManagement {
```

Comment:
is there a check that these two fields cannot be true at the same time?

Reply:
No, can't see this check anywhere. Same for DBaaS,VMCache, Alerting..

Reply:
Here it is)
https://github.com/percona/pmm-managed/blob/461cb41817f9838d7fde755ce799e1211eb22ac5/models/settings_helpers.go#L269-L281

Reply:
:facepalm: I was looking on to it before... thx